Frontend/app/class/detail/detail.component.ts

```diff
@@ -126,4 +127,23 @@ export class ClassDetailComponent implements OnInit, OnDestroy {
     this.toastr.error('Unable to join class');
   }
 
+  leaveClass(classCode) {
+    this.isLeaveClassInProgress = true;
+    this.userService.leaveClass(classCode)
+    .subscribe(
+      data => this.handleLeaveSuccess(data.json().payload),
+      err => this.handleLeaveError(err)
+    );
+  }
+
+  handleLeaveSuccess(json) {
+    this.isLeaveClassInProgress = false;
+    this.toastr.success('You have left ' + json.title);
```

Comment:
I'm not changing the response object.. so I just removed the class name and now its "Successfully unenrolled"